### PR TITLE
[Image Classification] Measure the performance of session.run() in ORT rather than classifier() of Transformer.js

### PR DESCRIPTION
**Pull request by ibelem**
This Image Classification demo is based on Transformer.js, which experimentally enabled WebNN by modifying some source code of Transformer.js.

```
const classifier = await pipeline('image-classification', modelPath, options);
const url = 'tiger.jpg';
const output = await classifier(url);
```

Image Classification:

1. Swapped the original perf.html and index.html
2. Removed the new perf.html

- Now the index.html will measure the performance of session.run() in ORT, the same as other demos.
- The index.html will not measure the performance of Transformer.js classifier() function anymore.
- User can change the test run numbers as usual.

@Adele101 PTAL

### CODE REVIEW

File: demos/image-classification/index.js [hunk 1]
```diff
@@ -237,49 +208,40 @@ const checkWebNN = async () => {
...
+      label_uploadImage.setAttribute('class', 'disabled');
       uploadImage.disabled = true;
       classify.disabled = true;
     }
   }
 };
 
 const initModelSelector = () => {
-  if (getQueryValue("provider") && getQueryValue("devicetype")) {
+  if(getQueryValue("provider") && getQueryValue("devicetype")) {
```
Comment (fdwr) [inline demos/image-classification/index.js:219]:
```suggestion
  if (getQueryValue("provider") && getQueryValue("devicetype")) {
```

File: demos/image-classification/index.js [hunk 2]
```diff
@@ -194,7 +166,7 @@ const checkWebNN = async () => {
   ) {
     circle.setAttribute("class", "none");
     info.innerHTML = "";
-  }
+  }
```
Comment (fdwr) [inline demos/image-classification/index.js:169]:
```suggestion
  }
```

File: demos/image-classification/index.js [hunk 3]
```diff
@@ -16,14 +13,15 @@ import {
   updateQueryStringParameter,
   getMedian,
   getAverage,
+  getMinimum,
   asyncErrorHandling,
-  getMode,
+  getMode
```
Comment (fdwr) [inline demos/image-classification/index.js:18]: (minor) Isn't it usually better to retain trailing commas in import lists to reduce the diff noise when adding/removing/reordering parameters?

File: demos/image-classification/index.html [hunk 4]
```diff
@@ -88,7 +88,8 @@ <h1 class="title">
         <div class="grid-2">
             <div>
                 <div id="container">
-                    <img id="image" src="./static/tiger.jpg" />
+                    <img id="image"
+                        src="./static/tiger.jpg" />
```
Comment (fdwr) [inline demos/image-classification/index.html:92]:
```suggestion
                    <img id="image" src="./static/tiger.jpg" />
```
(minor) random new line break?

File: demos/image-classification/index.html [hunk 5]
```diff
@@ -146,28 +140,33 @@ <h1 class="title">
...
                                 <td id="first"></td>
+                                <td class="r">Best Inference</td>
+                                <td id="best"></td>
                             </tr>
                             <tr>
-                                <td class="r">Average</td>
+                                <td class="r">Average Inference</td>
                                 <td id="average"></td>
+                                <td class="r">Median Inference</td>
+                                <td id="median"></td>
```
Comment (fdwr) [inline demos/image-classification/index.html:163]: 👍 These metrics are pretty complete (first, best, average, median...).

**Review by fdwr — APPROVED** [review]
Thanks Belem - minor polish comments. I think Adele has permissions to merge.

File: demos/image-classification/index.js (hunk 1 above)
Comment (ibelem) [reply demos/image-classification/index.js]: Fixed.

File: demos/image-classification/index.js [hunk 6]
```diff
@@ -205,27 +177,26 @@ const checkWebNN = async () => {
       status.setAttribute("class", "green");
       info.innerHTML = "WebNN supported";
 
-      if (deviceType.toLowerCase() === "npu") {
+      if(deviceType.toLowerCase() === 'npu') {
```
[the remark this answers is not in the document]
Comment (ibelem) [reply demos/image-classification/index.js]: Fixed.

File: demos/image-classification/index.js (hunk 2 above)
Comment (ibelem) [reply demos/image-classification/index.js]: Fixed.

File: demos/image-classification/index.js (hunk 3 above)
Comment (ibelem) [reply demos/image-classification/index.js]: Correct, thanks @fdwr for pointing this out. :)

File: demos/image-classification/index.html (hunk 4 above)
Comment (ibelem) [reply demos/image-classification/index.html]: Fixed :)

**Comment by ibelem** [comment]
Thanks @fdwr 's review, nearly all the format issues have been fixed. PTAL

**Review by fdwr — APPROVED** [review]

**Pull request merged**